replica-identity-full-performance.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "On the performance of REPLICA IDENTITY FULL in Postgres"
```

Comment:
```suggestion
title: "Improve the performance of REPLICA IDENTITY FULL in Postgres"
```

Reply:
We're not really improving it, just discussing it.